Backport jetpack 15.0-a.5 Changes (#44823)
Automattic/jetpack: projects/packages/videopress/CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.32.1] - 2025-08-18
### Added

> - Added type-check placeholder to implement it later. [#44795]

Reviewer: ```suggestion
- Add type-check placeholder script (to be implemented). [#44795]
```
